src/main/webapp/homepage_script.js

```diff
@@ -221,11 +221,15 @@ function getUserNeighborhood() {
 /* Function that returns a promise to get and return the user's location */
 function getUserLocation() {
     return new Promise((resolve, reject) => {
-        if (navigator.geolocation) {
-            navigator.geolocation.getCurrentPosition(function(position) {
-                var location = {lat: position.coords.latitude, lng: position.coords.longitude};
-                resolve(location);
-            }, function(err) {
+        navigator.geolocation.getCurrentPosition(function(position) {
```

Comment:
Do we know that `navigator.geolocation` always exists?

Reply:
My bad, I should've kept that if-statement. I wrongly thought `function(err)` would be able to handle those cases. I'll update this in the next commit.